tokens.py

```diff
@@ -8,55 +8,35 @@ def auto():
 
 class TokenTypes(Enum):
     # keywords
-    SELECT   = auto()
-    FROM     = auto()
-    EDGES    = auto()
-    NODES    = auto()
-    TRAVERSING = auto()
-    BY       = auto()
-    DEPTH    = auto()
-    TOPOLOGICAL = auto()
-    WHERE    = auto()
-    LIMIT    = auto()
+    KW    = auto() # SELECT, FROM, EDGES, NODES, TRAVERSE, BY, DEPTH, BREADTH, WHERE, LIMIT
     
-    # booleans
-    TRUE     = auto()
-    FALSE    = auto()
-
     # parentheses
-    LPAR     = auto()
-    RPAR     = auto()
+    LPAR  = auto()
+    RPAR  = auto()
+
+    # comma
+    COMMA = auto()
 
-    # logical operators
-    NOT  = auto()
+    # operators
+    OP    = auto() # AND, OR, NOT, >, <, >=, <=, =, <>, +, -, /, *, ||
     
-    # other operators
-    GT    = auto()  # >
-    LT    = auto()  # <
-    GTEQ  = auto()  # >=
-    LTEQ  = auto()  # <=
-    EQ    = auto()  # = 
-    LTGT  = auto()  # <>
-    PLUS  = auto()  # +
-    MINUS = auto()  # -
-    STAR  = auto()  # *
-    SLASH = auto()  # /
-    DBAR  = auto()  # ||
- 
     # types and identifiers
-    NUM    = auto()
-    STR    = auto()
-    ID     = auto()
+    BOOL  = auto()
+    NUM   = auto()
+    STR   = auto()
+    ID    = auto()
     
     # end of line / end of file
-    EOL    = auto()
-    EOF    = auto()
+    EOL   = auto()
+    EOF   = auto()
 
 class Token(object):
```

Comment:
You don't need to specify the parent if the parent is `object`, `class Token:` will do fine.

Reply:
I originally did this in Python 2, but now that's it's Python 3 I'll fix this.